exercises/practice/bob/.approaches/if-statements/content.md

```diff
@@ -0,0 +1,65 @@
+# `if` statements
+
+```javascript
+export function hey(message) {
+  const speech = message.trimEnd();
+  if (speech == '') return 'Fine. Be that way!';
+
+  const isQuestion = speech.endsWith('?');
+  const isShout = /[A-Z]{1}/.test(speech) && speech == speech.toUpperCase();
+
+  if (isShout)
+    return isQuestion
+      ? "Calm down, I know what I'm doing!"
+      : 'Whoa, chill out!';
+  if (isQuestion) return 'Sure.';
+  return 'Whatever.';
+}
+```
+
+In this approach you have a series of `if` statements using the private methods to evaluate the conditions.
+As soon as the right condition is found, the correct response is returned.
+
+Note that there are no `else if` or `else` statements.
+If an `if` statement can return, then an `else if` or `else` is not needed.
+Execution will either return or will continue to the next statement anyway.
```

Comment:
Maybe put this in a note block?

````suggestion
```exercism/note
Note that there are no `else if` or `else` statements.
If an `if` statement can return, then an `else if` or `else` is not needed.
Execution will either return or will continue to the next statement anyway.
```
````

Reply:
We didn't in C#, but it's a good idea.